src/app/features/home/home.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, inject, OnInit } from '@angular/core';
+import { Job } from '../jobs/models/job';
+import { JobService } from '../jobs/service/job.service';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrl: './home.component.scss',
+})
+export class HomeComponent implements OnInit {
```

Comment:
remove on init

Reply:
Constructor is predefined default method of the typescript class. There is no relation between Angular and constructor. Normally we use constructor to define/initialize some variables, but when we have tasks related to Angular's bindings we move to Angular's ngOnInit life cycle hook. ngOnInit is called just after the constructor call. We can also do the same work in the constructor but its preferable to use ngOnInit to start Angular's binding.

Reply:
@PrettyJuice Thanks for the clarification!